lib/traject/sul_config.rb

```diff
@@ -27,6 +27,21 @@
   accumulator << settings['aspace_config_set']
 end
 
+to_field 'sul_ark_link_ssi, sul_ark_id_ssi' do |record, _accumulator, context|
```

Comment:
`to_field` in traject is meant to define rules for accumulating entries for a single field. Something like this would be more idiomatic Traject:

```
to_field 'sul_ark_link_ssi', extract_xpath('/ead/archdesc/did/unitid[@type="ark"]/extref', to_text: false) do |_record, accumulator|
  accumulator.map! do |dao|
    dao.attributes['href']
  end
end

to_field 'sul_ark_id_ssi', extract_xpath('/ead/archdesc/did/unitid[@type="ark"]/extref', to_text: false) do |_record, accumulator|
  accumulator.map! do |dao|
    dao.attributes['href'].text.gsub('https://archives.stanford.edu/', '')
  end
end
```